drivers/tee/optee/optee_msg.h

```diff
@@ -414,5 +414,5 @@ struct optee_msg_arg {
  *					above
  */
 #define OPTEE_MSG_RPC_CMD_SHM_FREE	7
-
+#define OPTEE_MSG_RPC_CMD_BENCH_REG	20
```

Comment:
Add a short description above, with same formating as for the other command IDs.